app/controllers/users/two_factor_authentication_setup_controller.rb

```diff
@@ -44,7 +44,7 @@ def backup_code_only_processing
     # rubocop:disable Metrics/MethodLength
     def process_valid_form
       case @two_factor_options_form.selection
-      when 'sms', 'voice'
+      when 'phone'
```

Comment:
Given that 'sms' and 'voice' are still valid options according to the form, what if we kept those and added `phone`? Otherwise I think this will 500 if you change the option to `sms` or `voice`.

Reply:
Similarly, we should probably stick `phone` behind `sms` in the options controller

Reply:
done and done